# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

Header header
string type
EventRegion event_region
TrackedObject tracked_object
